android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```
@@ -241,9 +227,137 @@ void disposeTrack(String id) {
 
     void switchCamera(String trackId) {
         TrackPrivate track = tracks.get(trackId);
-        if (track != null && track.videoCaptureController != null) {
-            track.videoCaptureController.switchCamera();
+        if (track != null && track.videoCaptureController instanceof CameraVideoCaptureController) {
+            CameraVideoCaptureController videoCaptureController = (CameraVideoCaptureController) track.videoCaptureController;
+            videoCaptureController.switchCamera();
+        }
+    }
+
+    public void getDisplayMedia(Promise promise) {
+        if (this.promise != null) {
+            promise.reject(new RuntimeException("Another operation is pending."));
+            return;
+        }
+
+        Activity currentActivity = this.reactContext.getCurrentActivity();
+        if (currentActivity == null) {
+            promise.reject(new RuntimeException("No current Activity."));
+            return;
+        }
+
+        this.promise = promise;
+
+        MediaProjectionManager mediaProjectionManager =
+            (MediaProjectionManager) currentActivity.getApplication().getSystemService(
+                Context.MEDIA_PROJECTION_SERVICE);
+        currentActivity.startActivityForResult(
+            mediaProjectionManager.createScreenCaptureIntent(), PERMISSION_REQUEST_CODE);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
```

What about M? O even L? We should return an error at least.